build/git_revision.go

```diff
@@ -30,8 +30,13 @@ var (
 // GitRevision installs a particular git revision by cloning
 // the repository and building it per product BuildInstructions
 type GitRevision struct {
-	Product      product.Product
-	InstallDir   string
+	Product    product.Product
+	InstallDir string
+
+	// LicenseDir represents directory path where to install license files (required for enterprise versions, optional for OSS)
```

Comment:
```suggestion
	// LicenseDir represents directory path where to install license files.
```

Enterprise versions don't seem relevant in the context of `build.GitRevision{}` as the consumer would have to have access to the source code which is not publicly available. Relatedly there is no `Enterprise` field in this struct.